.github/workflows/test-r-scripts.yaml

```diff
@@ -23,6 +23,9 @@ jobs:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
+
+      - uses: r-lib/actions/setup-pandoc@master
```

Comment:
I'll follow up with a different PR to setup LaTeX. But notice how the pandoc part now works

https://github.com/2DegreesInvesting/PACTA_analysis/pull/291/checks?check_run_id=1333759327